letta/services/source_manager.py

```diff
@@ -0,0 +1,108 @@
+from typing import List, Optional
+
+from letta.orm.errors import NoResultFound
+from letta.orm.organization import Organization as OrganizationModel
+from letta.orm.source import Source as SourceModel
+from letta.schemas.embedding_config import EmbeddingConfig
+from letta.schemas.source import Source as PydanticSource
+from letta.schemas.source import SourceCreate, SourceUpdate
+from letta.schemas.user import User as PydanticUser
+from letta.utils import enforce_types, printd
+
+
+class SourceManager:
+    """Manager class to handle business logic related to Sources."""
+
+    # This is used when no embedding config is provided
+    DEFAULT_EMBEDDING_CONFIG = EmbeddingConfig(
```

Comment:
I think we should avoid having defaults (they've caused a lot of issues in the past) - we should just require it in `SourceCreate` and use that

Reply:
ack